### DISCUSSION TITLE
( RTP Packet Lost ) A guide on how to fix this issue when using windows and webrtc + obs (WHIP)

### DISCUSSION THREAD

**Discussion by vertexgamer** [bluenviron/mediamtx#3737] [General]
1- Open the windows registry editor
2- Go to **Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\AFD\Parameters**
3- Add 2 DWORDs and name them **DefaultReceiveWindow** and **DefaultSendWindow**
4- Assign to them a DECIMAL value of **131072** or a value multiple of 4096
5- Restart windows

N.B. the value above worked well with UDP 35mbps streaming, but dropped a few packets at 100mbps. Also there has been a slight decrease in tcp performance with multiple connections (before: 2300mbps down & 670mbps up) (now: 2200mbps down & 650mbps up)

**Comment by vertexgamer** [comment]
EDIT: i decided to increase the value to 23068672 (16MB) to achieve a localhost UDP bandwidth of 18gbps with no packet loss (tested with iperf3)

**Comment by jaffe-fly** [comment]
how to set in linux
